### DISCUSSION TITLE
Inbound Parse get MessageId

### DISCUSSION THREAD

**Discussion by JannyM85** [Jericho/StrongGrid#374] [Q&A]
Hello, it si possible to get message id from inbound parse? In my scenario I send email with dynamic template and i need receive send email because there are solved dynamic data. When i get the inbound email i need replace Subject and Body of my original email.  It si possible to do that with inboudn webhook, or is there eny other posibilities? Thanks!

**Comment by Jericho** [comment]
To my knowledge the payload of the webhook received from SendGrid does not contain a "message id" property. They include a dozen or so properties such as `to`, `from`, `text`, `html`, `sender_ip`, `spam_report`, etc.

**Reply by JannyM85** [reply]
I found the message id in header. Name of header is Message-ID.  For example Message-ID: <<9FoZK46JRSyu8LcKEY779A@geopod-ismtpd-6-0>>. 
The first part of string withotu start char < is message id. If you parse it you can use it and find original message.

**Reply by Jericho** [reply]
I'm glad you fond what you were looking for.
